src/components/AddFundsInterstitial.js

```diff
@@ -86,13 +86,14 @@ const Subtitle = styled(Title)`
   margin-top: ${({ isSmallPhone }) => (isSmallPhone ? 19 : 42)};
 `;
 
-const AmountText = styled(Text).attrs({
+const AmountText = styled(Text).attrs(({ children }) => ({
   align: 'center',
+  children: android ? ` ${children.join('')} ` : children,
```

Comment:
Is there a reason for the leading & trailing spaces?

Reply:
Oh I see... That's the hack for the shadows to not get cut off

Reply:
yes,